lib/cuckoo/core/startup.py

```diff
@@ -424,12 +425,16 @@ def init_routing():
                     f"The routing table that has been configured ({routing.routing.rt_table}) for dirty line interface is not available"
                 )
 
-        # Disable & enable NAT on this network interface. Disable it just
-        # in case we still had the same rule from a previous run.
-        rooter("disable_nat", routing.routing.internet)
-        rooter("enable_nat", routing.routing.internet)
-
-        # Populate routing table with entries from main routing table.
+        if routing.routing.nat:
+            # Disable & enable NAT on this network interface. Disable it just
+            # in case we still had the same rule from a previous run.
+            rooter("disable_nat", routing.routing.internet)
+            rooter("enable_nat", routing.routing.internet)
+            # Populate routing table with entries from main routing table.
+        else:
+            rooter("disable_nat", routing.routing.internet)
+        if routing.routing.no_local_routing:
+            rooter("init_vrf", routing.routing.rt_table, routing.routing.internet)
```

Comment:
Hi @piolug93 
now that is on master we are having some conectivity issues, shouldn't no_local_routing logic be inside the else?
```
else:
    rooter("disable_nat", routing.routing.internet)
    if routing.routing.no_local_routing:
        rooter("init_vrf", routing.routing.rt_table, routing.routing.internet)
```

Reply:
What problem do you have ? What you have set in routing.conf for nat and no_local_routing ?

Reply:
It's good observation. That should be in else block in accordance with comment from routing.conf for no_local_routing value.

Reply:
@doomedraven  thx for fix. 
https://github.com/kevoreilly/CAPEv2/commit/925ba96aba58057b1d1bae119063fdf41bc0c506

Reply:
Thank you both